uber_sante/services/availability_service.py

```diff
@@ -0,0 +1,60 @@
+from uber_sante.utils.dbutil import DBUtil
+from uber_sante.models.appointment import WalkinAppointment, AnnualAppointment
+
+
+class AvailabilityService:
+
+    def __init__(self):
+        self.db = DBUtil.get_instance()
+
+    def get_availabilities(self, schedule_request):
+
+        select_stmt = 'SELECT * FROM Availability WHERE year = ? AND month = ? AND day = ?'
+
+        date = schedule_request.get_request_date()
```

Comment:
check schedule request object for the type of request: if monthly, the day attribute should be wildcar, if day request, month, day, and year should be used in the query